build-lib/util/licenser.js

```diff
@@ -25,7 +25,7 @@ Licenser.prototype.constructor = Licenser;
 
 Licenser.prototype.build = function () {
 
-  const fileDescriptions = [].concat(...this.inputPaths.map(dirname => {
```

Comment:
i mean on CI. Wondered why., installed 4.2.2 locally and got same. I just checked and babel register is on for versions less than `v4.0.0`